mapmatcher/src/main/java/io/github/nik9000/mapmatcher/ListMatcher.java

```diff
@@ -62,7 +75,14 @@ private ListMatcher(List<Matcher<?>> matchers) {
    *         expected followed by the provided item
    */
   public ListMatcher item(Object value) {
-    return item(matcherFor(value));
+    return item(matcherFor(value, false));
+  }
+
+  /*
+  * Apply the extraOk to all children
+  * */
+  public ListMatcher item(Object value, Boolean extraOk) {
```

Comment:
Probably wants to be `boolean` instead of `Boolean`. Also the method probably wants to be javadoc.

Here's a question - if you pass a matcher here and it happens to be a `MapMatcher` then should the parameter `extraOk` apply? Probably not. But it's good to mention it.